frontend/packages/dev-console/src/actions/add-resources.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import {
+  GitAltIcon,
+  OsImageIcon,
+  CatalogIcon,
+  CubeIcon,
+  DatabaseIcon,
+} from '@patternfly/react-icons';
+import { ImportOptions } from '../components/import/import-types';
+import { KebabAction, createKebabAction } from '../utils/add-resources-menu-utils';
+
+export const fromGit = createKebabAction('From Git', <GitAltIcon />, ImportOptions.GIT);
+export const containerImage = createKebabAction(
```

Comment:
nit: can we add an empty line between each `export`.
Not a big deal since prettier accepts it.